src/mammooc-course-evaluation.html

```diff
@@ -183,6 +192,10 @@
                 super.ready();
             }
 
+            _generateCourseUrl(provider, courseId) {
+                return 'https://mammooc-dev.herokuapp.com/courses/' + provider + '~' + courseId;
```

Comment:
I am probably just missing something, but why does this contain a dev environment's URL?

Reply:
At this point, it is intended. The PR was against the dev branch, where we use the dev environment. The master branch contains the same line with the production URL: https://github.com/mammooc/mammooc-rating-widget/blob/master/src/mammooc-course-evaluation.html#L196